Sources/Core/State/WaitingNetworkState.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  WaitingNetworkState.swift
+//  ModernAVPlayer
+//
+//  Created by raphael ankierman on 18/05/2018.
+//
+
+import AVFoundation
+import Foundation
+
+public final class WaitingNetworkState: PlayerState {
+    
+    // MARK: - Inputs
+    
+    public unowned var context: PlayerContext
```

Comment:
what's the point to have an unowned?
```swift
public let context: PlayerContext
```
looks enough for me. I see every state has that pattern, but I can't remember why... 🤔

Reply:
concreteplayercontext has a state reference 
all state has a concreteplayercontext
that's why

Reply:
ok